packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -565,19 +563,34 @@ export const selectMinAllowedInputWithFee = memoizeWithArgs(
     },
 );
 
-export const selectIsCoinjoinBlockedByAmountsTooSmall = memoizeWithArgs(
-    (state: CoinjoinRootState, accountKey: AccountKey) => {
-        const selectedAccount = selectSelectedAccount(state);
-
-        const minAllowedInputWithFee = selectMinAllowedInputWithFee(state, accountKey);
-        const targetAnonymity = selectCurrentTargetAnonymity(state) || 0;
-
-        const anonymitySet = selectedAccount?.addresses?.anonymitySet || {};
-        const utxos = selectedAccount?.utxo || [];
-
-        // return true if all non-privat funds are too small
+export const selectAreUtxosTooSmallByAccount = memoizeWithArgs(
+    (state: CoinjoinRootState, account: Account) => {
```

Comment:
other selectors select by `accountKey`, why not this one?

Reply:
To avoid nesting another selector as it needs to access `account`. Not sure about the best approach here.

Reply:
I would say it's not important to pass the whole thing, a key is enough for the hook to get all required data and return a result :)